packages/nouns-webapp/src/hooks/useBlockTimestamp.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
+import { useEthers } from '@usedapp/core';
+
+export function useBlockTimestamp(blockno: number | undefined): number | undefined {
+  const { library } = useEthers();
+  const [blockTimestamp, setBlockTimestamp] = useState<number | undefined>();
+
+  useEffect(() => {
+
+    async function updateBlockTimestamp() {
+      if (!blockno) return;
```

Comment:
We use the term `blockNumber` in other places in the repo. Please change instances of `blockno` to that.